server/__test__/user.test.js

```diff
@@ -1,9 +1,16 @@
+import mongoose from 'mongoose';
+
 const request = require('supertest');
 const app = require('../app.js');
 const actionStatus = require('../constants/actionStatus');
 
 describe('User Test', () => {
   describe('User Signup', () => {
+    afterAll(async (done) => {
```

Comment:
You mean delete data from collections?
I am using Mongo InMemoryServer which I thought won't need cleanup, otherwise should I get all collections and drop them?